ci/test/00_setup_env_arm.sh

```diff
@@ -25,4 +25,4 @@ export RUN_FUNCTIONAL_TESTS=false
 export GOAL="install"
 # -Wno-psabi is to disable ABI warnings: "note: parameter passing for argument of type ... changed in GCC 7.1"
 # This could be removed once the ABI change warning does not show up by default
-export BITCOIN_CONFIG="--enable-reduce-exports CXXFLAGS=-Wno-psabi"
+export BITCOIN_CONFIG="--enable-external-signer --enable-reduce-exports CXXFLAGS=-Wno-psabi"
```

Comment:
Is it possible to add this to `BITCOIN_CONFIG_ALL` and then opt out in the windows build with `--disable-external-signer`?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/24397#issuecomment-1047748269).